apps/sms/views/conversation/js/conversation.js

```diff
@@ -407,6 +407,7 @@ var ConversationView = {
   },
 
   getIdIterator: function conv_getIdIterator() {
+    // TODO: Will need to replace with Conversation service method.
```

Comment:
Well, it might depend on where we cache the messages map. It was cached in Threads originally, so it's possible to have a method in Conversation service to get the messages (iterator). Or you prefer to cache the messages in conversation view?